sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/ModelsRepositoryClientBuilder.java

```diff
@@ -75,21 +73,25 @@
     /**
      * The public constructor for ModelsRepositoryClientBuilder
      */
-    public ModelsRepositoryClientBuilder()
-    {
+    public ModelsRepositoryClientBuilder() {
         additionalPolicies = new ArrayList<>();
         properties = CoreUtils.getProperties(MODELS_REPOSITORY_PROPERTIES);
         httpLogOptions = new HttpLogOptions();
+        try {
+            this.repositoryEndpoint = new URI(ModelsRepositoryConstants.DEFAULT_MODELS_REPOSITORY_ENDPOINT);
+        } catch (URISyntaxException e) {
+            // We know it won't throw since it's a known endpoint and has been validated.
```

Comment:
If this is known and won't throw could we turn the URI into a constant somewhere?

Reply:
done.